cargo-pgrx/src/command/install.rs

```diff
@@ -353,17 +353,13 @@ pub(crate) fn find_library_file(
     manifest: &cargo_toml::Manifest,
     build_command_messages: &Vec<cargo_metadata::Message>,
 ) -> eyre::Result<PathBuf> {
-    let crate_name = if let Some(ref package) = manifest.package {
-        &package.name
-    } else {
-        return Err(eyre!("Could not get crate name from manifest."));
-    };
+    let target_name = manifest.target_name()?;
 
     let mut library_file = None;
     for message in build_command_messages {
         match message {
             cargo_metadata::Message::CompilerArtifact(artifact) => {
-                if artifact.target.name != *crate_name {
+                if artifact.target.name != *target_name {
```

Comment:
`artifact.target.name` is filled with crate name as-is (ie `pgrx-tests`) or lib name, depending on whether lib name has a name override or not, but `cargo_manifest::Manifest` auto fills lib name with crate name with hyphen replaced with underscore (ie `pgrx_tests`). Added `target_name` on `PgrxManifestExt` to return a value matching what `cargo_metadata` expects